derive/examples/generic-trait-bounds.rs

```diff
@@ -16,23 +16,23 @@ pub trait Rpc<One, Two, Three>
 
 	/// Adds two numbers and returns a result
 	#[rpc(name = "setTwo")]
-	fn set_two(&self, _: Two) -> Result<()>;
+	fn set_two(&self, a: Two) -> Result<()>;
 
 	#[rpc(name = "getThree")]
 	fn get_three(&self) -> Result<Three>;
 
 	/// Performs asynchronous operation
 	#[rpc(name = "beFancy")]
-	fn call(&self, _: One) -> FutureResult<(One, u64), Error>;
+	fn call(&self, a: One) -> FutureResult<(One, u64), Error>;
 }
 
 struct RpcImpl;
 
 #[derive(Serialize, Deserialize)]
 struct InAndOut { foo: u64 }
-#[derive(Deserialize)]
+#[derive(Serialize, Deserialize)]
```

Comment:
This should be only required when client is generated. Could you please alter this example to not do that so that we are sure that generic parameters bounds are relaxed?

Maybe create another example that uses the client and requires the parameters to be ser-de